package.json

```diff
@@ -45,7 +45,8 @@
     "webpack": "^3.3.0"
   },
   "dependencies": {
-    "leap-core": "^0.20.3",
+    "babel-plugin-transform-builtin-extend": "^1.1.2",
```

Comment:
what is it for?

Reply:
Don't know, it was missing when trying to build